pywinauto/application.py

```diff
@@ -626,15 +631,27 @@ def print_identifiers(ctrls, current_depth=1):
                     criteria_texts.append(u'control_type="{}"'.format(control_type))
                 if title or class_name or auto_id:
                     output += indent + u'child_window(' + u', '.join(criteria_texts) + u')'
-                if six.PY3:
-                    print(output)
+
+                if log_file is None:

```

Comment:
Looks like too many `if`s for me. How about one `if` and re-using something like `log_func` callable object (`log_func=print` by default).